### ISSUE TITLE
[Task]: Update copy on the Add-on reporting form to make it clear how to report a copyright/trademark infringment.

### ISSUE THREAD

**Issue by abyrne-moz** [mozilla/addons#1772]
### Description

The add-on reporting form does not currently clearly explain how to submit a copyright or trademark infringement report, causing some reports to be incorrectly filed. 

We need to modify the add-on reporting form, so the line under "It violates the law or contains content that violates the law" just says "Example: Fraud."  If possible, we should also add a parenthesis or asterisk next to the example, that says "If you wish to report a copyright or trademark infringement, you can learn more about how to do so by clicking here: https://www.mozilla.org/en-US/about/legal/report-infringement/."  This content has been drafted by our trust and safety team.

This should help our users get a more timely and accurate response to their DMCA query.

### Acceptance Criteria

```[tasklist]
  ### Acceptance Criteria
  - [ ] Add-on reporting form contains the copy in the ticket.
  ```

### Checks

- [X] If the issue is ready to work on, I have removed the "needs:info" label.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/AMO2-57)

**Comment by eviljeff** [comment]
> "If you wish to report a copyright or trademark infringement, you can learn more about how to do so by clicking here: https://www.mozilla.org/en-US/about/legal/report-infringement/."

@abyrne-moz can you rephrase T&S's copy without the "clicking here: <url>" at the end? (google search for "click here" for the many articles decrying it 😬)

**Comment by abyrne-moz** [comment]
"If you wish to report a copyright or trademark infringement, you can learn more about how to do so in our [Copyright or Trademark Infringement Reporting article](https://www.mozilla.org/en-US/about/legal/report-infringement/)."

**Comment by ioanarusiczki** [comment]
could be something with deploys because I looked on -dev and the change is not there yet

**Comment by diox** [comment]
Yes, addons-frontend builds are broken, will be fixed soon, watch the slack channel for updates.

**Comment by diox** [comment]
Fixed now.

**Comment by ioanarusiczki** [comment]
This is available for all entities reported not only addons or themes but I assume it's expected.

![violates the law](https://github.com/mozilla/addons/assets/33448286/201d46ec-9d9d-4bcc-af4c-e1c9ad78a7de)
